### PR TITLE
Fixes NCR not showing up in latejoin menu, gives faction positions prefixes.

### PR DESCRIPTION
**Pull request by uomo91**
## About The Pull Request

~~Somebody (I think myself) added "NCR" to the beginning of their job titles, this was causing them to be left out of the latejoin proc.~~
By request I've just given every position in every faction, except wastelanders, a prefix for their faction.

## Why It's Good For The Game

Bugs bad, convenience good.

### CODE REVIEW

**Comment by Hekzder** [comment]
I'd prefer the prefix be left, I'm not sure exactly what the issue is but if we can solve it and keep the prefix that would be ideal.

**Comment by uomo91** [comment]
> I'd prefer the prefix be left, I'm not sure exactly what the issue is but if we can solve it and keep the prefix that would be ideal.

The only place this is visible is on the job preferences menu.
And if we do this for the NCR, we should do it for all the jobs.

**Comment by uomo91** [comment]
Oops.

File: mojave/jobs/job_types/legion/primelegionary.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/job/ms13/legion/primelegionary
-	title = "Prime Legionary"
+	title = "Legion Prime"
```
Comment (Hekzder) [inline mojave/jobs/job_types/legion/primelegionary.dm:2]:
```suggestion
	title = "Prime Legionary"
```

File: mojave/jobs/job_types/legion/recruitlegionary.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /datum/job/ms13/legion/recruitlegionary
-	title = "Recruit Legionary"
+	title = "Legion Recruit"
```
Comment (Hekzder) [inline mojave/jobs/job_types/legion/recruitlegionary.dm:2]:
```suggestion
	title = "Recruit Legionary"
```

File: mojave/jobs/job_types/legion/veteranlegionary.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /datum/job/ms13/legion/veteranlegionary
-	title = "Veteran Legionary"
+	title = "Legion Veteran"
```
Comment (Hekzder) [inline mojave/jobs/job_types/legion/veteranlegionary.dm:2]:
```suggestion
	title = "Veteran Legionary"
```

**Review by Hekzder — CHANGES_REQUESTED** [review]
Prefixes are good, the base Legionary jobs don't really need the prefix, since they have Legion/Legionary in the name by default. 

Recruit Legionary, Prime Legionary, etc. sounds much better than "Prime Legion", while still making it pretty obvious that you're playing as the Legion.

**Review by Hekzder — APPROVED** [review]
This should be fine to merge now, once we get the CI stuff sorted.

**Pull request merged**